code/core/src/main/scala/com.snowplowanalytics.stream/loader/utils/CredentialsLookup.scala

```diff
@@ -52,11 +52,15 @@ object CredentialsLookup {
       } else if (isIam(a) && isIam(s)) {
         InstanceProfileCredentialsProvider.getInstance()
       } else if (isIam(a) || isIam(s)) {
-        throw new RuntimeException("access-key and secret-key must both be set to 'iam', or neither")
+        throw new RuntimeException(
+          "access-key and secret-key must both be set to 'iam', or neither"
+        )
       } else if (isEnv(a) && isEnv(s)) {
         new EnvironmentVariableCredentialsProvider()
       } else if (isEnv(a) || isEnv(s)) {
-        throw new RuntimeException("access-key and secret-key must both be set to 'env', or neither")
+        throw new RuntimeException(
+          "access-key and secret-key must both be set to 'env', or neither"
```

Comment:
I think this type of error is a little not descriptive enough, we need to say which variable is the culprit rather than guess

Reply:
@aahmedg This the older code ,I haven`t changed this,changes are due to scala formatting.Let me see how I can make it more descriptive